docs/getting-started/prefect-quickstart.md

```diff
@@ -0,0 +1,167 @@
+# Prefect Quickstart
+
+This guide is designed to show you how to deploy a Prefect flow in as few steps as possible. For a more comprehensive introduction to Prefect's core components and how they work together, please follow our [tutorial](/tutorial/index/).
+
+### Step 1: [Install Prefect](/getting-started/installation/)
+
+<div class="terminal">
+```bash
+pip install -U prefect
+```
+</div>
+
+
+### Step 2: Connect to Prefect's API
+
+Sign up for a forever free [Prefect Cloud Account](/cloud/) or, alternatively, host your own [server](/host/) with a subset of Prefect Cloud's features.
+
+1. To sign in with an existing account or register an account, go to [https://app.prefect.cloud/](https://app.prefect.cloud/).
+2. [Create a workspace](#create-a-workspace) for your account.
+3. Use the `prefect cloud login` Prefect CLI command to log into Prefect Cloud from your environment.
+
+    <div class="terminal">
+    ```bash
+    prefect cloud login
+    ```
+    </div>
+
+### Step 3: Author a Flow
```

Comment:
```suggestion
### Step 3: Create a flow
```
Anything not in a top level heading is sentence case and Prefect concepts don't get special capitalization, according to the style guide, I believe.